scalyr_agent/json_lib/parser.py

```diff
@@ -638,11 +645,14 @@ def __match(self, chars, error_message):
         and consume them.  In case of a mismatch, raise an exception.
         Only supports low-ASCII characters.
 
-        If error_message is None, we generate a default message."""
+        If error_message is None, we generate a default message.
```

Comment:
Why type is `chars` here?  Can you add documentation for that.

Reply:
The type is `six.text_type`(unicode) here. I think it was confusing because I did not write docstring correctly.